### PR TITLE
Task #3517 Changed template design on SSH keys pages

### PR DESCRIPTION
**Pull request by siarheipuhach**
Check pages:

/hosting/ssh_keys
/hosting/create_ssh_key
/hosting/ssh_keys_choice
Those pages where changed (ssh_keys_choice was added).
add_key
key_choice
key_list
ssh key list_mobile

### CODE REVIEW

File: hosting/templates/hosting/choice_ssh_keys.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                                 {% csrf_token %}
+                                 <button type="submit" class="btn btn-success choice-button generate-btn">
+                                     {% trans "Generate"%}
+                                 </button>
+                             </form>
+
+                         </div>
+                         <div class="choice-container-line"></div>
+                         <div class="right-choice-container">
+                             <h3>I want to use my existing public key.
```
Comment (pcoder) [inline hosting/templates/hosting/choice_ssh_keys.html]: We seem to be missing the translate block for the text here?

File: hosting/templates/hosting/user_key.html [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+{% block content %}
 <div>
 	<div class="virtual-machine-container dashboard-container ">
 		<div class="row">
 			<div class="container-table col-md-9 col-md-offset-2">
 				 <div  class="col-sm-12">
                     <form method="POST" action="" novalidate class="form-ssh">
                         {% csrf_token %}
-				        <h3><i class="fa fa-key fa-separate" aria-hidden="true"></i>{% trans "Access Key"%} </h3>
+				        <h3><i class="fa fa-key fa-separate" aria-hidden="true"></i>{% trans "Add you public SSH key"%} </h3>
```
Comment (pcoder) [inline hosting/templates/hosting/user_key.html]: you -> your

**Comment by verysanghee** [comment]
@siarheipuhach server error 500 on both firefox and chrome on key page

**Comment by verysanghee** [comment]
@siarheipuhach Before anything the style is very much off.  Btn alignment and the "?" position.
screen shot 2017-07-07 at 22 04 50

**Comment by verysanghee** [comment]
screen shot 2017-07-07 at 22 01 54
"+" : Lato Bold

**Comment by verysanghee** [comment]
@tiariwav can you update this PR with above comments and bring this PR into usable state?

**Comment by verysanghee** [comment]
reproposed as #419 by @tiwariav so will close this PR.

**Pull request closed without merging**